services/121-service/src/registration/registration.entity.ts

```diff
@@ -76,11 +76,14 @@ export class RegistrationEntity extends CascadeDeleteEntity {
   @Column({ type: 'integer', nullable: true })
   public inclusionScore: number | null;
 
-  @ManyToOne((_type) => FinancialServiceProviderEntity)
-  @JoinColumn({ name: 'fspId' })
-  public fsp: FinancialServiceProviderEntity;
+  // ##TODO find out if this can also be required instead of optional
+  @ManyToOne((_type) => ProgramFinancialServiceProviderConfigurationEntity)
```

Comment:
Why do we need this relationship? Kobo sends this, so it becomes a value in RegistrationAttributeData, right?

Reply:
We drew that relationship in Miro right? We can use it to join ProgramFinancialServiceProviderConfigurationEntity stuff to registrations. Like the label or the name or the programFspConfigProperties

Reply:
Yes, that is handy.